portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -1135,14 +1135,62 @@ definition {
 	}
 
 	@description = "Verify that a Text field's Autocomplete Options can be translated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test TextFieldAutocompleteOptionsCanBeTranslated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133497 TextFieldAutocompleteOptionsCanBeTranslated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Text");
+
+		Form.gotoAutocompleteTab();
+
+		FormViewBuilder.editAutoComplete(key_optionValue = "Manually");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Apple",
+			optionValue = "1");
+
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Manzana",
+			optionValue = "1");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
```

Comment:
Hi @piresdm! Just a question about this line. Is it necessary, since it's not described on the steps of the `ticket`? In case it's not needed, could you remove it, please? Thanks =)

Reply:
Sorry, I've found that this line is really necessary. =)